src/tfr_control/src/drivebase.cpp

```diff
@@ -0,0 +1,27 @@
+/****************************************************************************************
+ * File:            drivebase.cpp
+ * 
+ * Purpose:         This node converts Twist data from the /cmd_vel topic to angular
+ *                  velocity of the two motor sets (left and right). Most of the
+ *                  responsibilities are delegated to the DrivebasePublisher class.
+ * 
+ * Launched By:     drivebase.launch
+ ***************************************************************************************/
+#include "ros/ros.h"
+#include "drivebase_publisher.h"
+
+
+int main(int argc, char **argv)
+{
+    ros::init(argc, argv, "drivebase");
+
+    ros::NodeHandle n;
+    float axle_length, wheel_radius;
+    n.getParam("axle_length", axle_length);
+    n.getParam("wheel_radius", wheel_radius);
```

Comment:
I think we should either add default values here (which can be done with a slightly different method: `n.param<float>("axle_length", axle_length, [some default value])`), or have a check to make sure that the values actually get loaded and is non-zero.

Reply:
Possibly add a call to ROS_WARN as well if the values aren't loaded and it uses the defaults?

Reply:
I agree.  What I'm not sure about is how severe such an error should be.  If the values don't come in, or come in wrong, the robot won't move as expected but it won't ruin everything.  That said, it should be easy for us to always make sure our inputs are always loaded.

Reply:
If we set the defaults to what we'd probably be feeding it anyways, I think warning or even info (depending on how much we care).